config/holes.toml

```diff
@@ -735,6 +735,38 @@ preamble = '''
     for example <b>15</b> should print <b>FizzBuzz</b>.
 '''
 
+['Forsyth-Edwards Notation']
+category = 'Transform'
```

Comment:
We could also use `Gaming` as we lack many holes in that category, what do you think?